samples/splitter/splitter.cpp

```diff
@@ -116,11 +118,18 @@ class MyFrame: public wxFrame
     void OnUpdateUIUnsplit(wxUpdateUIEvent& event);
     void OnUpdateUIInvisible(wxUpdateUIEvent& event);
 
+    bool IsSashLocked() const { return m_lockSash; }
+
+    void SetSashPos(int pos) { m_sashPos = pos; }
+    int GetSashPos() const { return m_sashPos; }
+
 private:
     wxWindow *m_left, *m_right;
 
     wxSplitterWindow* m_splitter;
     wxWindow *m_replacewindow;
+    int m_sashPos;
+    bool m_lockSash:1;
```

Comment:
Minor, but there is really no need to optimize this here, so
```suggestion
    bool m_lockSash;
```

Reply:
Fixed with the next commit.